cob_twist_controller/CMakeLists.txt

```diff
@@ -24,29 +24,29 @@ include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN_INCLUDE_DIRS} ${oroco
 
 
 ## Build
+set(SRC_C_DIR "src/constraint_solvers")
+set(SRC_CS_DIR "${SRC_C_DIR}/solvers")
+
 # Declare a cpp library
+add_library(damping_methods src/damping_methods/damping_base.cpp src/damping_methods/damping.cpp)
+add_library(constraint_solvers ${SRC_CS_DIR}/constraint_solver_base.cpp ${SRC_CS_DIR}/unconstraint_solver.cpp ${SRC_CS_DIR}/wln_joint_limit_avoidance_solver.cpp ${SRC_CS_DIR}/weighted_least_norm_solver.cpp ${SRC_C_DIR}/constraint_solver_factory_builder.cpp)
```

Comment:
fine with me...let's keep the solver files as they are until we get a better idea of what the solver structure will look like after null-space-projection and task-augmentation is added